lib/elixir/lib/option_parser.ex

```diff
@@ -499,4 +562,28 @@ defmodule OptionParser do
   defp negative_number?(arg) do
     match?({_, ""}, Float.parse(arg))
   end
+
+  defp format_errors(errors, opts) do
+    details = Enum.map_join(errors, ". ", &format_error(&1, opts))
+    total = Enum.count(errors)
+    error = if total == 1, do: "error", else: "errors"
+    "#{total} #{error} found! #{details}"
```

Comment:
I think I mentioned this a while back, but I'm 👎 on the exclamation mark. Now, we even have [this recent commit](https://github.com/elixir-lang/elixir/commit/d22e456047bab2375c960474edb7288ef53a0750#diff-ed0a878a29a4e0266095d1153c4f3a00L31) that removed the `!` from `IO.warn` in favour of a more sober colon :).